jung-visualization/src/main/java/edu/uci/ics/jung/visualization/spatial/rtree/Node.java

```diff
@@ -51,7 +51,7 @@
 
   Collection<Shape> collectGrids(Collection<Shape> list);
 
-  Collection<T> getVisibleElements(Shape shape);
+  Collection<T> getVisibleElements(Collection<T> visibleElements, Shape shape);
```

Comment:
minor request (not related to this line): please rename this class RTreeNode.  I don't want people to get it confused with a graph node.